app/examples/circleci/install-docker-tools.yml

```diff
@@ -0,0 +1,31 @@
+version: 2.1
+
+orbs:
+  docker: circleci/docker@2.4.0
+
+jobs:
+  install-docker-tools:
+    docker:
+      - image: cimg/base:stable
+    steps:
+      - checkout
+      - docker/install-docker-tools:
+          install-docker: true
+          docker-install-dir: /usr/local/bin
```

Comment:
if you are going to specify an option it would make more sense to use a value that is not the same as the default one to be able to tell if it is not being respected